<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>616</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1082</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_10">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>214</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="downPushButton">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>55</y>
       <width>65</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftPushButton">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>30</y>
       <width>50</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightPushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>62</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upPushButton">
     <property name="geometry">
      <rect>
       <x>106</x>
       <y>4</y>
       <width>65</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acwPushButton">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>81</y>
       <width>65</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>ACW</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cwPushButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>81</y>
       <width>65</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>CW</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>65</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="moveRadioButton">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rotateRadioButton">
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="zoomRadioButton">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="framePointColor">
             <property name="maximumSize">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonPointColor">
             <property name="text">
              <string>Point Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frameLineColor">
             <property name="maximumSize">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonLineColor">
             <property name="text">
              <string>Line Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frameBFColor">
             <property name="maximumSize">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonBGColor">
             <property name="text">
              <string>Background Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_9">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSpinBox" name="spinBoxLineWidth">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxPointWidth">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="PointWlabel">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Point Width</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="lineWlabel">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Line Width</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="solidRadioButton">
          <property name="text">
           <string>Line</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBoxRound">
          <property name="text">
           <string>Round</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="pointRadioButton">
          <property name="text">
           <string>Point</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="strippleRadioButton">
          <property name="text">
           <string>Stripple</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="boxOrtho">
          <property name="text">
           <string>Parallel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="saveAsGif">
          <property name="text">
           <string>Save as gif</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsBmp">
          <property name="text">
           <string>Save as bmp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsJpeg">
          <property name="text">
           <string>Save as jpeg</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>340</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/3d-logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="open">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>150</y>
      <width>219</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="File_name">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_vertices">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edges:</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_edges">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
